engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -739,6 +871,35 @@ else if (propertyName.equals(entry.getKey())) {
   }
 
   @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES + "/{moduleName}/properties")
```

Comment:
I have removed /moduleName/properties api and reused the /operatorName/properties to list the module properties.